doc/install.asciidoc

```diff
@@ -357,6 +357,29 @@ standard location for your distro (`/usr/share/applications` and
 The normal `setup.py install` doesn't install these files, so you'll have to do
 it as part of the packaging process.
 
+[[python]]
+Running qutebrowser with Python
```

Comment:
I'm still opposed to this section as a whole, sorry. I don't see a reason why this should be used instead of using `tox` to create a virtualenv, and the `sudo pip install` thing is very bad advice (it messes with your system's python, with a very real risk of breaking things).

Can you elaborate on why you think this is needed, rather than telling people to create a virtualenv via tox and use that?